src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -120,32 +115,25 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 	
                 	// HH Account should be created if:
                     //    1. The account isn't set by the user
-                    //    2. The Account Model is set to Household or Administrative Account
-                    if (CAO_Constants.isHhOrAdmAccountModel() && c.AccountId == null) {
+                    //    2. The Account Model is not blank
+                    if (c.AccountId == null && defaultRecTypeID != null) {
                         listContactNeedAccount.add(c);
                     }
                     
-                    // grab the Accounts that need to have the newly assigned Contact Id to them if:
+                    // grab the Accounts that need to have the newly assigned Contact Id added to them if:
                     //    1. If the contact is connected to an Account
-                    //    2. The Contact is flagged as an individual
-                    //    3. we are in the one-to-one or HH Account model
                     if (c.AccountId != null) { 
                         // contacts are connected to Accounts, make the connection in the other direction
                         mapAccountIdContactId.put(c.AccountId, c.Id);
-                    }
-
-	                // HHAccount renaming needs to be called to set various HH naming fields like greeting.
-	                if (c.AccountId != null && (c.Account.RecordType.Name == CAO_Constants.HH_ACCOUNT_PROCESSOR
```

Comment:
We don't yet have custom naming (we have an open work item for that). Standard account naming happens in https://github.com/SalesforceFoundation/HEDAP/blob/feature/216-acc-rec-types-change/src/classes/ACCT_IndividualAccounts_TDTM.cls#L480-L500